Package.swift

```diff
@@ -38,17 +38,25 @@ let package = Package(
     // a module or a test suite.
     // Targets can depend on other targets in this package, and on products
     // in packages which this package depends on.
+    .target(
+      name: "CombineShim",
+      dependencies: [.product(
+        name: "OpenCombine",
+        package: "OpenCombine",
+        condition: .when(platforms: [.wasi, .linux])
```

Comment:
Could you swap out the `OpenCombine` url in `.package` on line 34 when on Linux?

Reply:
I don't think so, at least after looking at [the proposal](https://github.com/apple/swift-evolution/blob/master/proposals/0273-swiftpm-conditional-target-dependencies.md). I'll try to submit my changes to the upstream OpenCombine repo, I just have another old open PR there with the `zip` implementation that I need to finish first...